components/script/dom/webxr/xrviewerpose.rs

```diff
@@ -175,7 +175,11 @@ impl XRViewerPose {
         let transform: RigidTransform3D<f32, Viewer, BaseSpace> =
             viewer_pose.transform.then(&to_base);
         let transform = XRRigidTransform::new(global, cast_transform(transform), can_gc);
-        let pose = reflect_dom_object(Box::new(XRViewerPose::new_inherited(&transform)), global);
+        let pose = reflect_dom_object(
+            Box::new(XRViewerPose::new_inherited(&transform)),
+            global,
+            CanGc::note(),
```

Comment:
there is already a `can_gc` argument for this function, so no need to call `CanGc::note`